pages/en/publish/testing-digital-goods-api.md

```diff
@@ -5,27 +5,30 @@ date: 2021-11-05
 - **Digital Goods and other payment options are served correctly -** If a user enters a digital storefront from an app context, they should see Google Play Billing available when going to subscribe. If the user enters the payment flow from a browser context, they should see the alternative payment flows served.
 - **[Out of App Purchase](https://chromeos.dev/en/publish/pwa-play-billing#out-of-app-purchases) Flows**
   - **Subscription from promo code -** Subscriptions come in as unacknowledged purchases. When the user subscribes and opens the app, the subscription should be verified, entitlements should be granted, and the subscription should be acknowledged.
-  - **One time purchase from promo code -** One time purchases should be acknowledged but entitlements may not be granted. Once the user logs into the app, make sure that the entitlement is granted.
-  - **Consumable from promo code -** Consumable purchases should be acknowledged but entitlements may not be granted. Once the user logs into the app, make sure that the entitlement is granted and then consume the purchase so the user can repurchase the item.
+  - **One time purchase from promo code -** One time purchases will already be acknowledged by Play, but the user won't have the entitlement yet. Once the user logs into the app, make sure that the entitlement is granted.
+  - **Consumable from promo code -** Consumable purchases will already be acknowledged by Play, but the user won't have the entitlement yet. Once the user logs into the app, make sure that the entitlement is granted and then consume the purchase so the user can repurchase the item.
   - **Cancel subscription from store account management -** When the user manages their subscription outside of your application (such as from the store page) make sure that their subscription is canceled correctly in your backend and entitlements are not continuing to be granted.
   - **Pause subscription from store account management -** The user can also manage their subscription from outside the application and [pause their subscription](https://support.google.com/googleplay/answer/7018481). The benefits of the subscription should be suspended during the paused period and resumed when the pause has ended.
   - **Resubscribe from store management**
     - If canceling and resubscribing right away, the cancelation should just be undone.
-    - If canceling and waiting for the subscription to expire and then resubscribing from the store management page, this would be considered an out of app purchase and would need to be acknowledged in app.
+    - If canceling and waiting for the subscription to expire and then resubscribing from the store management page, this would be considered an out of app purchase and would need to be acknowledged in app. There will be a confirm subscription message in red text in the Play Store Subscription managmenet page if the subscription is not acknowledged.
```

Comment:
```suggestion
    - If canceling and waiting for the subscription to expire and then resubscribing from the store management page, this would be considered an out of app purchase and would need to be acknowledged in app. There will be a confirm subscription message in red text in the Play Store Subscription management page if the subscription is not acknowledged.
```